From b95adbb2eab7cc0738e7c805bb33a3848cdbd54b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 04 Oct 2019 11:48:58 +0300
Subject: [PATCH] [SW-0] gaudi: pll: switch hbm to 800mhz

800MHz has been found as the most reliable HBM frequency for now.
Switch the HBM PLL to 800MHz.
We will get higher frequencies later.

Change-Id: I714f06e703a9be1dfda9ef73ac948aff9ea50e2c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index c6a4eaf..78189b0 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -177,7 +177,7 @@
 		},
 	}, {
 		.name = "HBM", .base = mmPSOC_HBM_PLL_BASE,
-		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
+		.nr = 0, .nf = 63, .od = 3, .nb = 31, /* 800 MHz */
 		.div_factor = { 0, 1, -1, -1 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_DIV_PLL,
@@ -219,7 +219,7 @@
 		},
 	}, {
 		.name = "NIC2_HBM", .base = mmNIC2_HBM_PLL_BASE,
-		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
+		.nr = 0, .nf = 63, .od = 3, .nb = 31, /* 800 MHz */
 		.div_factor = { 0, 1, -1, -1 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_DIV_PLL,
